src/main/scala/edu/ie3/util/geo/CoordinateDistance.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.util.geo
+
+import org.locationtech.jts.geom.Point
+
+import java.util.Objects
+import javax.measure.quantity.Length
+import tech.units.indriya.ComparableQuantity
+
+/** Wraps two coordinates with the distance between the first one and the second
+  * one, can be compared by distance to another CoordinateDistance
+  * @param from
+  *   The first coordinate
+  * @param to
+  *   The second coordinate
+  * @param distance
+  *   The distance from A to B
+  */
+class CoordinateDistance private (
```

Comment:
I don't quite understand in what circumstances it would make sense to wrap the inputs and outputs of such a function in a class, but maybe you can enlighten me